resources/dicts/thoughts/alive/deputy.json

```diff
@@ -0,0 +1,69 @@
+[
+    {
+        "id": "deputy_dead_deputy",
+        "thoughts": [
+            "Wishes they could speak to r_c to learn how to assign patrols better"
+        ],
+        "random_status_constraint": [
+            "deputy"
+        ]
```

Comment:
Isn't there a 'live status' check?
I think there are missing some conditions, the main cat should also be a deputy, but the 'main_status_constraint' is missing